docs/gcc_incompatibilities.md

```diff
@@ -0,0 +1,201 @@
+# GCC - Clang system headers incompatibilities
+
+## GCC fixincludes
```

Comment:
```md
## <a name="gcc-fixincludes"></a> GCC fixincludes
```

Reply:
Why we need the anchors. Github handles the links automatically, see this readme for example: https://github.com/martong/finstrument_mock/blob/master/README.md

Reply:
Cool :+1: I don't know about this feature.